bin/example_component.jsx.txt

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+import "./COMPONENT_NAME.less";
```

Comment:
Agreed, I'll add a comment about stateless components. I'm not sure what the doc format should be (jsdoc?) so I'll leave that til we know more